Model/Sync/Catalog/Processor/Main.php

```diff
@@ -371,30 +371,31 @@ protected function getApiParams(
         array $yotpoData,
         array $parentIds,
         array $parentData,
-        $visibleVariants = false
+        $isVisibleVariant
```

Comment:
$isVisibleVariant = false should be there by default. Otherwise it will be needed to pass a value always.

Reply:
@manaftryzens it is being called only once on `Model/Sync/Catalog/Processor.php` and in [that case](https://github.com/YotpoLtd/magento2-module-core/blob/47637c75df2cdb3648347fd434de711b432a18a8/Model/Sync/Catalog/Processor.php#L265), the last argument is being passed.
Why is it needed to have a default value?

Reply:
Please ignore if it is being passed. In this case we don't need a default value.